src/Arcus.Messaging.Tests.Core/EventHubs/Extensions/EventDataExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Arcus.Messaging.Abstractions.EventHubs;
+using GuardNet;
+
+// ReSharper disable once CheckNamespace
+namespace Azure.Messaging.EventHubs
+{
+    public static class EventDataExtensions
```

Comment:
Added this to the test project so that the current tests are all generating the job ID (not interesting for many tests).